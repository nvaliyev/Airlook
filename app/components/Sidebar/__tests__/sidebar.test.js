import 'react-native';
import React from './node_modules/react';
import ShallowRenderer from './node_modules/react-test-renderer/shallow';
import DrawerContent from '..';
describe(' SideBar content', () => {
    test(' SideBar content snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(< DrawerContent/>);
        expect(snapshot).toMatchSnapshot();
    });
});


