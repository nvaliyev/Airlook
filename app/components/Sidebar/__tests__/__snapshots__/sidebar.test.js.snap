// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` SideBar content  SideBar content snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F7F9FC",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#F7F9FC",
        "justifyContent": "center",
        "maxHeight": 70,
        "width": "100%",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../app/assets/img/logo.png",
        }
      }
      style={
        Object {
          "maxHeight": 250,
          "resizeMode": "contain",
          "width": 200,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignContent": "center",
        "backgroundColor": "#F7F9FC",
        "flex": 1,
        "flexDirection": "column",
        "height": "80%",
        "justifyContent": "flex-start",
        "marginTop": 20,
      }
    }
  >
    <DrawerButtons
      ico={
        <Icon
          allowFontScaling={false}
          color="#6f347c"
          name="calendar"
          size={25}
        />
      }
      onPress={[Function]}
      text="Flights"
    />
    <DrawerButtons
      ico={
        <Icon
          allowFontScaling={false}
          color="#6f347c"
          name="cog"
          size={25}
        />
      }
      onPress={[Function]}
      text="Settings"
    />
    <DrawerButtons
      ico={
        <Icon
          allowFontScaling={false}
          color="#6f347c"
          name="newspaper-o"
          size={25}
        />
      }
      onPress={[Function]}
      text="News"
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 2,
          "justifyContent": "flex-end",
        }
      }
    >
      <DrawerButtons
        ico={
          <Icon
            allowFontScaling={false}
            color="#6f347c"
            name="logout"
            size={25}
          />
        }
        onPress={[Function]}
        text="Log Out"
      />
    </View>
  </View>
</View>
`;
